spec/models/concerns/hmrc_interface_resultable_spec.rb

```diff
@@ -366,6 +366,93 @@
     end
   end
 
+  describe "#start_and_end_dates_for_employments" do
+    subject(:start_and_end_dates_for_employments) { instance.start_and_end_dates_for_employments }
+
+    context "when multiple employments exist" do
+      let(:result) do
+        {
+          "data" => [
+            { "use_case" => "use_case_one" },
+            {
+              "employments/paye/employments": [
+                {
+                  "endDate": "2099-12-31",
+                  "startDate": "2023-01-26"
+                },
+                {
+                  "endDate": "2022-11-11",
+                  "startDate": "2022-09-11"
+                }
+              ]
+            },
+          ]
+        }
+      end
+
+      it "returns a multiline String with \"start-date to end-date\"" do
+        expect(start_and_end_dates_for_employments).to eql("2023-01-26 to 2099-12-31\n2022-09-11 to 2022-11-11")
```

Comment:
Would this handle if HMRC reverse the start/end date sequence?  Could you set up the data as 
```
               {
                  "endDate": "2099-12-31",
                  "startDate": "2023-01-26"
                },
                {
                  "startDate": "2022-09-11",
                  "endDate": "2022-11-11"
                }
```
To test?

Reply:
thank Colin. It should handle the different positions of `startDate` and `endDate` as it is a key lookup, but will do a quick test. 
```
employments&.map do |emp|
  "#{emp&.fetch("startDate")} to #{emp&.fetch("endDate")}"
end
```
What concerns me more is reliance on the ordering for this, and more significantly where we are selecting the top/first payment/income (previous PR) - but caseworker says she always sees payments chronological order, most recent first.

Reply:
have added a test for above and it goes green
https://github.com/ministryofjustice/laa-assure-hmrc-data/pull/293/commits/0ff7b24ead427f8e3e0f59c354d09524efcb03b4